CONTRIBUTING.md

```diff
@@ -0,0 +1,44 @@
+# How to contribute
+
+Reporting bugs
+--------------
+
+Perhaps the easiest way to contribute to Qtile is to report any bugs you
+run into on the [github issue tracker](https://github.com/qtile/qtile/issues).
+
+Useful bug reports are ones that get bugs fixed. A useful bug report normally
+has two qualities:
+
+1. **Reproducible.** If your bug is not reproducible it will never get fixed.
+   You should clearly mention the steps to reproduce the bug. Do not assume or
+   skip any reproducing step. Described the issue, step-by-step, so that it is
+   easy to reproduce and fix.
+
+2. **Specific.** Do not write a essay about the problem. Be Specific and to the
+   point. Try to summarize the problem in minimum words yet in effective way.
+   Do not combine multiple problems even they seem to be similar. Write
+   different reports for each problem.
+   
```

Comment:
You might want to add a sentence here about how adding logs from `~/.qtile.log` or an xtrace [1] is helpful for diagnosing and fixing bugs.

[1] http://qtile.readthedocs.org/en/latest/manual/hacking.html#capturing-an-xtrace